Assets/Scripts/EOSLobbyManager.cs

```diff
@@ -2185,7 +2187,7 @@ private void AddLocalUserAttributes()
         /// <param name="JoinLobbyCompleted">Callback when join lobby is completed</param>
         public void JoinLobby(string lobbyId, LobbyDetails lobbyDetails, bool presenceEnabled, OnLobbyCallback JoinLobbyCompleted)
         {
-
+            
```

Comment:
This is a pedantic request, but could you delete the whitespace that was added here?

Reply:
Cleaned up